webapp/pages/profile/_id/_slug.vue

```diff
@@ -507,7 +460,11 @@ export default {
         return UserQuery(this.$i18n)
       },
       variables() {
-        return { id: this.$route.params.id }
+        return {
+          id: this.$route.params.id,
+          followedByCount: this.followedByCount,
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-03-31T12:04:04Z" title="Tuesday, March 31st 2020, 2:04:04 pm +02:00">Mar 31, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -507,7 +459,11 @@ export default {
         return UserQuery(this.$i18n)
       },
       variables() {
-        return { id: this.$route.params.id }
+        return {
+          id: this.$route.params.id,
+          followedByCount: this.followedByCount || 7,
+          followingCount: this.followingCount || 7,
```


do we need to have both a default in `data` and a fallback here?

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-01T07:40:55Z" title="Wednesday, April 1st 2020, 9:40:55 am +02:00">Apr 1, 2020</time>_

---


no, we actually don't
removed in 9dd65aa